runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java

```diff
@@ -426,8 +427,17 @@ public static SpritePixels getImageSpritePixels(BufferedImage image, Client clie
 
 		try
 		{
-			new PixelGrabber(image, 0, 0, image.getWidth(), image.getHeight(), pixels, 0, image.getWidth())
-				.grabPixels();
+			PixelGrabber g = new PixelGrabber(image, 0, 0, image.getWidth(), image.getHeight(), pixels, 0, image.getWidth());
+			g.setColorModel(new DirectColorModel(32, 0xff0000, 0xff00, 0xff, 0xff000000));
+			g.grabPixels();
+
+			for (int i = 0; i < pixels.length; i++)
```

Comment:
What's this doing? Why is it in this commit?

Reply:
It ensures any fully transparent pixels are full black, so the sprite draw routines draw them as transparent